rai/secure.cpp

```diff
@@ -10,8 +10,6 @@
 
 #include <ed25519-donna/ed25519.h>
 
-thread_local CryptoPP::AutoSeededRandomPool rai::random_pool;
```

Comment:
This is removed because it's unused?

Reply:
@cryptocode It's in ria_lib, so it's a duplicate symbol.

Reply:
This is here because of how windows does symbols in a shared library.  Need to make sure Windows still builds if this is removed.